start.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+#
+# Start script for cdn-analyser
+
+export APP_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+PROXY_ARGS=""
+
+if [[ -z "${MESOS_SLAVE_PID}" ]]; then
+    echo "Vagrant not supported, please use Docker"
+
+else
+
+    PORT="$1"
+    CONFIG_URL="$2"
+    ENVIRONMENT="$3"
+    APP_NAME="$4"
+
+    source /etc/profile
+ß
+    echo "Downloading environment from: ${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}"
+    wget -O "${APP_DIR}/private_env" "${CONFIG_URL}/${ENVIRONMENT}/private_env"
+    wget -O "${APP_DIR}/global_env" "${CONFIG_URL}/${ENVIRONMENT}/global_env"
+    wget -O "${APP_DIR}/app_env" "${CONFIG_URL}/${ENVIRONMENT}/${APP_NAME}/env"
+    source "${APP_DIR}/private_env"
+    source "${APP_DIR}/global_env"
+    source "${APP_DIR}/app_env"
+
+fi
+
+exec java ${JAVA_MEM_ARGS} -jar ${PROXY_ARGS} -Dserver.port="${PORT}" -Dspring.data.mongodb.uri="$MONGODB_URL" "${APP_DIR}/cdn-analyser.jar"
```

Comment:
I think this will need tweaking at least a little bit. At the moment it looks like it's attempting to pull config from a url